src/contracts/core/StrategyManager.sol

```diff
@@ -439,7 +461,7 @@ contract StrategyManager is
         uint256[] memory shares = new uint256[](strategiesLength);
 
         for (uint256 i = 0; i < strategiesLength; ) {
-            shares[i] = stakerStrategyShares[staker][stakerStrategyList[staker][i]];
+            shares[i] = nonNormalizedStakerStrategyShares[staker][stakerStrategyList[staker][i]];
```

Comment:
i believe this is only used by the UI and should be normalized

Reply:
ah, good point. perhaps we want a new `getNonNormalizedDeposits` function, and we keep the behavior of `getDeposits` (returning normalized shares) the same as it was before.

Reply:
addressed in this commit https://github.com/Layr-Labs/eigenlayer-contracts/pull/543/commits/a48492a849dea0c48eacd46f353f36168a7fd213
we need to decide which functions to keep and which to scrap, or else we'll end up with both non- and normalized versions of everything